devices/slv.js

```diff
@@ -0,0 +1,23 @@
+const extend = require('../lib/extend');
+
+module.exports = [
+    {
+        fingerprint: [
+            {modelID: '1001248'},
+            {modelID: 'ZBT-ColorTemperature-Panel'},
+        ],
+        model: '1001248',
+        vendor: 'SLV',
+        description: 'VALETO CCT LED Driver',
+        extend: extend.light_onoff_brightness_colortemp({colorTempRange: [153, 370]}),
+    },
+    {
+        zigbeeModel: ['1002994'],
+        model: '1002994',
+        vendor: 'SLV',
+        description: 'VALETO Remote',
+        fromZigbee: [],
```

Comment:
I would expect some commands here, does it send anything to the coordinator when pressing the buttons?

Reply:
Actually this a multi group remote delivered with the panels and I didn't work on i'ts integration more than just identifying it on the map (it's supposed th be directly binded to the LED panels).
Forget this device if you don't want to populate the file with a not yet really usefull description ;)